### DISCUSSION TITLE
Pass arbitrary command to service

### DISCUSSION THREAD

**Discussion by EricDriussi** [community/community#56676] [Actions] [answered]
### Select Topic Area

General

### Body

Hi there!
I'm migrating an old CircleCI pipeline to GH actions and need to pass an arbitrary command to the required mysql service.
In CircleCI one could do something like `command: [--secure-file-priv=]` while setting up the service, Is there an equivalent for GH actions?
Going by [this doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idservices) I understand It's not doable but I wanted to double check just in case.

**Comment by dgholz** [comment, marked as answer]
yeah, it's not doable. see #52675 for the most recent ask & https://github.com/orgs/community/discussions/26688#discussioncomment-3252877 for the first answer that said it's not possible.
